include/spdlog/sinks/wincolor_sink.h

```diff
@@ -0,0 +1,139 @@
+//
+// Copyright(c) 2016 Christopher J. Torres (a modified verison of ansicolor_sink).
+// Distributed under the MIT License (http://opensource.org/licenses/MIT)
+//
+
+#pragma once
+
+#include <spdlog/sinks/base_sink.h>
+#include <spdlog/common.h>
+
+#include <string>
+#include <map>
+
+namespace spdlog
+{
+namespace sinks
+{
+
+#ifndef _WIN32

```

Comment:
I think all the ```#ifdef _WIN32``` in this file can be omitted. It is called ```wincolor_sink``` after all. It should not be included by the wrapper in non windows